Broken test: test_get_current_columns_subset

This tests is complaining for me about the longitude of the response not matching expected. The response is a pretty odd value.

```
    @pytest.mark.slow
    def test_get_current_columns_subset(fake_coordinate):
        adapter = OpenMeteoAdapter()
        response = adapter.get_current(coordinate=fake_coordinate, past_days=1, forecast_days=1, columns=["temperature_2m"])
        assert response.status_code == 200
>       assert pytest.approx(110.0) == response.data["longitude"]
E       assert 110.0 ± 1.1e-04 == 110.03906
E         comparison failed
E         Obtained: 110.03906
E         Expected: 110.0 ± 1.1e-04
```

--- a/tests/forecasting/data_fetching_utilities/weather_provider/open_meteo/test_open_meteo_adapter.py
+++ b/tests/forecasting/data_fetching_utilities/weather_provider/open_meteo/test_open_meteo_adapter.py
@@ -50,7 +50,7 @@ def test_get_current_columns_subset(fake_coordinate):
     adapter = OpenMeteoAdapter()
     response = adapter.get_current(coordinate=fake_coordinate, past_days=1, forecast_days=1, columns=["temperature_2m"])
     assert response.status_code == 200
-    assert pytest.approx(110.0) == response.data["longitude"]
-    assert pytest.approx(30.0) == response.data["latitude"]
+    assert pytest.approx(110.0, rel=1e-2) == response.data["longitude"]
+    assert pytest.approx(30.0, rel=1e-2) == response.data["latitude"]
     assert len(response.data["hourly"]) == 2  # Time column and 'temperature_2m' column
     assert len(response.data["hourly_units"]) == 2

